app/callback.py

```diff
@@ -217,13 +218,18 @@ def get_cdc_template(
     exchange: Exchange = Exchange.BINANCE,
     current: bool = True) -> str:
     if exchange == Exchange.OKEX:
-        tickers = OkexAPI.get_usdt_tickers() if pair == "usdt" else OkexAPI.get_btc_tickers()
+        tickers = OkexAPI.get_usdt_tickers() if pair == Pairs.USDT else OkexAPI.get_btc_tickers()
     elif exchange == Exchange.BINANCE:
-        tickers = BinanceAPI.get_usdt_tickers() if pair == "usdt" else BinanceAPI.get_btc_tickers()
+        tickers = BinanceAPI.get_usdt_tickers() if pair == Pairs.USDT else BinanceAPI.get_btc_tickers()
     elif exchange == Exchange.FTX:
-        tickers = FtxAPI.get_usdt_tickers() if pair == "usdt" else FtxAPI.get_btc_tickers()
+        if pair == Pairs.USDT:
+            tickers = FtxAPI.get_usdt_tickers()
+        elif pair == Pairs.PERP:
+            tickers = FtxAPI.get_perp_tickers()
+        else:
+            tickers = FtxAPI.get_btc_tickers()
     elif exchange == Exchange.KUCOIN:
-        tickers = KucoinAPI.get_usdt_tickers() if pair == "usdt" else KucoinAPI.get_btc_tickers()
+        tickers = KucoinAPI.get_usdt_tickers() if pair == Pairs.USDT else KucoinAPI.get_btc_tickers()
```

Comment:
can we do something better. 

exchange = getExchangeAPI().

then. 

exchange.get_usdt_tickers() if pair == Pairs.USDT else exchange.get_btc_tickers()

abtracts `get_usdt_tickers` and `get_btc_tickers` across all exchanges.

create interface/abstract class for them.